### PR TITLE
replaced rejected with declined (nicer)

### LINKED ISSUE #6989: [CAL-1029] Rename "reject" to "decline"
feedback from a customer: "Reject" sounds kinda harsh, "decline" is better

<sub>[CAL-1029](https://linear.app/calcom/issue/CAL-1029/rename-reject-to-decline)</sub>

### PR DESCRIPTION
**Pull request by PeerRich**
from a customer: reject sounds kinda harsh, decline is better

### CODE REVIEW

**Comment by PeerRich** [comment]
changing the en/common.json will ask everyone in crowdin to update the string right?

File: apps/web/public/static/locales/en/common.json [hunk 1]
```diff
@@ -19,10 +19,10 @@
   "need_to_reschedule_or_cancel": "Need to reschedule or cancel?",
   "cancellation_reason": "Reason for cancellation (optional)",
   "cancellation_reason_placeholder": "Why are you cancelling?",
-  "rejection_reason": "Reason for rejecting",
-  "rejection_reason_title": "Reject the booking request?",
-  "rejection_reason_description": "Are you sure you want to reject the booking? We'll let the person who tried to book know. You can provide a reason below.",
-  "rejection_confirmation": "Reject the booking",
+  "rejection_reason": "Reason for declination",
```
Comment (baileypumfleet) [inline apps/web/public/static/locales/en/common.json:22]: Declination isn't really a word

File: apps/web/public/static/locales/en/common.json [hunk 2]
```diff
@@ -59,8 +59,8 @@
   "integration_meeting_id": "{{integrationName}} meeting ID: {{meetingId}}",
   "confirmed_event_type_subject": "Confirmed: {{eventType}} with {{name}} at {{date}}",
   "new_event_request": "New event request: {{attendeeName}} - {{date}} - {{eventType}}",
-  "confirm_or_reject_request": "Confirm or reject the request",
-  "check_bookings_page_to_confirm_or_reject": "Check your bookings page to confirm or reject the booking.",
+  "confirm_or_reject_request": "Confirm or decline the request",
```
Comment (baileypumfleet) [inline apps/web/public/static/locales/en/common.json:62]: If we're going for the usage of the word 'decline', should we change 'confirm' to 'accept'?

File: apps/web/public/static/locales/en/common.json [hunk 3]
```diff
@@ -1362,7 +1362,7 @@
   "fetching_calendars_error": "There was a problem fetching your calendars. Please <1>try again</1> or reach out to customer support.",
   "calendar_connection_fail": "Calendar connection failed",
   "booking_confirmation_success": "Booking confirmation succeeded",
-  "booking_rejection_success": "Booking rejection succeeded",
+  "booking_rejection_success": "Booking declination succeeded",
```
Comment (baileypumfleet) [inline apps/web/public/static/locales/en/common.json:1365]: Also here

**Review by pumfleet — CHANGES_REQUESTED** [review]
Left some comments about the words here

File: apps/web/public/static/locales/en/common.json (hunk 1 above)
Comment (PeerRich) [reply apps/web/public/static/locales/en/common.json:22]: https://www.dict.cc/?s=declination its not? 👀

File: apps/web/public/static/locales/en/common.json (hunk 2 above)
Comment (PeerRich) [reply apps/web/public/static/locales/en/common.json:62]: interesting

File: apps/web/public/static/locales/en/common.json (hunk 3 above)
Comment (PeerRich) [reply apps/web/public/static/locales/en/common.json:1365]: maybe 

Booking declined successfully?

File: apps/web/public/static/locales/en/common.json (hunk 1 above)
Comment (hariombalhara) [reply apps/web/public/static/locales/en/common.json:22]: Screenshot 2023-02-19 at 2 38 02 PM
Seems like a word with completely different meaning 😅

Comment (PeerRich) [reply apps/web/public/static/locales/en/common.json:22]: true!

**Comment by PeerRich** [comment]
lets keep reject for now

**Pull request closed without merging**